$NetBSD: patch-ac,v 1.4 2015/12/29 23:34:56 dholland Exp $

This is not the only scheme interpreter, so adjust the names in the
man page accordingly.

Honor PREFIX.

--- scheme.1.tmp.orig	Sun Jan 21 22:15:52 1996
+++ scheme.1.tmp	Thu Apr 12 19:05:35 2001
@@ -1,10 +1,10 @@
-.TH SCHEME 1  "Janaury 22, 1996" "local"
+.TH UMB-SCHEME 1  "Janaury 22, 1996" "local"
 .SH NAME
-scheme  \-  UMB Scheme  (Revision: 3.2 )
+umb-scheme  \-  UMB Scheme  (Revision: 3.2 )
 .SH ORIGIN
 University of Massachusetts at Boston
 .SH SYNOPSIS
-.B scheme [file ...]
+.B umb-scheme [file ...]
 .SH DESCRIPTION
 .ta +0.5i 2.0i 
 .PP
@@ -21,14 +21,14 @@
 .PP
 The following files are loaded in order at startup:
 .PP
-If the variable SCHEME_INIT is set in the user's environment by executing
+If the variable UMB_SCHEME_INIT is set in the user's environment by executing
 .nf
-        setenv SCHEME_INIT file
+        setenv UMB_SCHEME_INIT file
 .fi
 then file is loaded.
 .PP
-If SCHEME_INIT is not set and if a file 
-.I " .scheme "
+If UMB_SCHEME_INIT is not set and if a file 
+.I " .umb-scheme "
 exists in the user's home directory then it is loaded.
 .PP
 The files named as optional arguments are loaded from left to right.
@@ -99,11 +99,9 @@
 
 UMB Scheme can access Aubrey Jaffer's 
 .I "Scheme Portable Library (SLIB)."
-For documentation on SLIB see either
+For documentation on SLIB see
 .br
-.I "http://www.cs.indiana.edu/scheme-repository/SCM/slib_toc.html"
-or
-.I "/usr/local/lib/slib/slib.texinfo."
+.I "http://swissnet.ai.mit.edu/~jaffer/SLIB.html"
 
 .SH SYSTEM INTERFACE
 
@@ -281,7 +279,7 @@
 the debugger is turned 
 .I "off"
 by default.  One can insure the debugger is always turned on by putting a 
-call to debug in the Scheme Init file (e.g. .scheme).
+call to debug in the Scheme Init file (e.g. .umb-scheme).
 .PP
 Any scheme expression may be evaluated in debugging mode.  It is evaluated
 in the environment that existed when the break occurred in the top level
@@ -345,11 +343,11 @@
 UMB Scheme does not support nested debugging sessions.  (Keep it simple.)
 .fi
 .SH FILES
-/usr/local/lib/scheme/prelude.scheme
+@PREFIX@/share/umb-scheme/prelude.scheme
 .br
-/usr/local/lib/scheme/SLIB-for-umb-scheme.init
+@PREFIX@/share/umb-scheme/SLIB-for-umb-scheme.init
 .br
-/usr/local/lib/slib/*
+@PREFIX@/share/umb-scheme/slib/*
 
 .SH "SEE ALSO"
 Release Notes
